site/installation/README.md

```diff
@@ -27,6 +27,47 @@ Verify the version:
 $ kpt version
 ```
 
+## (Optional) enable shell auto-completion
```

Comment:
Can we move this to a separate PR and merge it after this change is released. Else the docs will be ahead of release as they are released with PR merge.

Reply:
Done. The docs change has been removed, and I will shortly create a new PR with just the docs change